R/lint.R

```diff
@@ -418,6 +430,12 @@ Lint <- function(filename, line_number = 1L, column_number = 1L, # nolint: objec
   obj
 }
 
+is_valid_range <- function(range, max_col) {
+  0L <= range[[1L]] &&
```

Comment:
maybe a `stopifnot(length(ranges) == 2L)`?

also why `range` vs `ranges` difference?

Reply:
oh nvm, I confused this with being the `check_ranges()` body